Resources/doc/index.rst

```diff
@@ -0,0 +1,45 @@
+Installation
+============
+
+Step 1: Download the Bundle
+---------------------------
+
+Open a command console, enter your project directory and execute the
+following command to download the latest stable version of this bundle:
+
+.. code-block:: bash
+
+    $ composer require symfony/monolog-bundle "~2.7"
+
+This command requires you to have Composer installed globally, as explained
+in the `installation chapter`_ of the Composer documentation.
+
+Step 2: Enable the Bundle
+-------------------------
+
+Then, enable the bundle by adding the following line in the ``app/AppKernel.php``
+file of your project:
```

Comment:
That would be more clear. I would suggest changing this in the Installation Instructions template as well: http://symfony.com/doc/current/cookbook/bundles/best_practices.html#installation-instructions